EU4toV2/Source/EU4World/EU4Country.cpp

```diff
@@ -107,27 +105,21 @@ EU4::Country::Country(
 			adjective = theAdjective.getString();
 		}
 	);
+	// This is obsolete and not applicable from at least 1.19+, probably further back
 	registerKeyword(std::regex("map_color"), [this](const std::string& unused, std::istream& theStream)
 		{
-			color = commonItems::Color(theStream);
+			auto colorColor = commonItems::Color(theStream);
+			colorColor.RandomlyFlunctuate(30);
 			// Countries whose colors are included in the object here tend to be generated countries,
 			// i.e. colonial nations which take on the color of their parent. To help distinguish 
 			// these countries from their parent's other colonies we randomly adjust the color.
-			color.RandomlyFlunctuate(30);
+			nationalColors.setMapColor(colorColor);
 		}
 	);
-	registerKeyword(std::regex("colors"), [this](const std::string& colorsString, std::istream& theStream)
+	registerKeyword(std::regex("colors"), [this, theVersion](const std::string& colorsString, std::istream& theStream)
```

Comment:
Looks like theVersion isn't being used, you can drop it from the []

Reply:
did a version switch first but then changed my mind and forced all to new structure. dropped.